test/small-integration-test.sh

```diff
@@ -44,7 +44,7 @@ if [ -n "${ALL_TESTS}" ]; then
     FLAGS=(
         "use_cache=${CACHE_DIR} -o ensure_diskfree=${ENSURE_DISKFREE_SIZE} -o fake_diskfree=${FAKE_FREE_DISK_SIZE}"
         enable_content_md5
-        enable_noobj_cache
+        disable_noobj_cache
```

Comment:
Isn't it better to test this part as enabled noobj cache?
Until now, it was inspected as enabled, so I would like to recommend this inspection.
I think that the test of disabled noobj cache is better to run in another test or not to run it.

Reply:
I changed the default so that no option specified = `-o enable_noobj_cache`.  Thus we should test the opposite or we can remove this option and speed up CI.

Reply:
I'm sorry, it was my misunderstanding.